frontend/src/app/main/ui/ds/product/milestone.cljs

```diff
@@ -0,0 +1,92 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns app.main.ui.ds.product.milestone
+  (:require-macros
+   [app.common.data.macros :as dm]
+   [app.main.style :as stl])
+  (:require
+   [app.common.data :as d]
+   [app.main.ui.ds.buttons.icon-button :refer [icon-button*]]
+   [app.main.ui.ds.controls.input :refer [input*]]
+   [app.main.ui.ds.foundations.assets.icon :as i]
+   [app.main.ui.ds.foundations.typography :as t]
+   [app.main.ui.ds.foundations.typography.text :refer [text*]]
+   [app.main.ui.ds.product.avatar :refer [avatar*]]
+   [app.main.ui.ds.utilities.date :refer [date* valid-date?]]
+   [app.util.i18n :as i18n :refer [tr]]
+   [rumext.v2 :as mf]))
+
+(def ^:private schema:milestone
+  [:map
+   [:class {:optional true} :string]
+   [:active {:optional true} :boolean]
+   [:editing {:optional true} :boolean]
+   [:autosaved {:optional true} :boolean]
+   [:versionToggled {:optional true} :boolean]
+   [:userName :string]
+   [:userAvatar {:optional true} [:maybe :string]]
+   [:userColor :string]
```

Comment:
It feels like this should be a `user` object with `name`, `avatar` and `color`, no?